apis/test_config.py

```diff
@@ -90,6 +89,31 @@ def name(self):
     return f'v{self.version}-{self.cores}'
 
 
+# We may want to use GKE in the future.
```

Comment:
Probably update it to a TODO comment? i.e. add GKE support for GPU. If you don't know who will implement the feature, feel free to assign it to me.